pkgs/applications/version-management/git-trim/default.nix

```diff
@@ -46,6 +47,7 @@ rustPlatform.buildRustPackage rec {
       IOKit
       CoreFoundation
       Security
+      zlib
```

Comment:
https://github.com/NixOS/nixpkgs/pull/369535#discussion_r1899880206. can make zlib unconditional ( and drop IOKit CoreFoundation and Security -- they are no-ops now as they are part of the stdenv)

Reply:
ah cool! will fixup - thanks for the callouts!

Reply:
When removing `Security` from the top level attrset, i get an error in eval - i guess because this is invoked with 
```nix
git-trim = darwin.apple_sdk_11_0.callPackage ../applications/version-management/git-trim {...}
```
in `all-packages.nix` - is that something i should swap out for the main `callPackage` now?

Reply:
I've made this ^ change and made sure it built